pyanaconda/payload/dnf/progress.py

```diff
@@ -0,0 +1,82 @@
+#
```

Comment:
I think progress and display should be together. Also the name doesn't really explain what it does.

Reply:
I can rename the files, but I don't see why they should be together. They are used at different phases of the installation.